flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -203,6 +212,7 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
         }
       },
     );
+    _currentRouteName = name;
```

Comment:
let's move this line into `_setCurrentRoute()` method, as that one is called on `didReplace()`, too

Reply:
done and also updated tests 👍